class.photon.php

```diff
@@ -290,6 +290,23 @@ public static function filter_the_content( $content ) {
 
 						// Supplant the original source value with our Photon URL
 						$photon_url = esc_url( $photon_url );
+						$zoom_url_2 = add_query_arg( 'zoom', 2, $photon_url );
+						$zoom_url_3 = add_query_arg( 'zoom', 3, $photon_url );
+						$zoom_url_4 = add_query_arg( 'zoom', 4, $photon_url );
+						$new_tag = str_replace( $src_orig, $photon_url, $new_tag );
```

Comment:
You're duplicating this line that is still down below at old line 293, new line 310.